backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQFixture.java

```diff
@@ -28,28 +28,23 @@
 
 import com.rabbitmq.client.AMQP;
 
-public class RabbitMQFixture {
-    public static final boolean DURABLE = true;
-    public static final boolean AUTO_ACK = true;
-    public static final AMQP.BasicProperties NO_PROPERTIES = null;
-    public static final String EXCHANGE_NAME = "exchangeName";
-    public static final String ROUTING_KEY = "routingKey";
-    public static final String DIRECT = "direct";
-    public static final boolean EXCLUSIVE = true;
-    public static final boolean AUTO_DELETE = true;
-    public static final String WORK_QUEUE = "workQueue";
+public interface RabbitMQFixture {
+    AMQP.BasicProperties NO_PROPERTIES = null;
```

Comment:
This should be in rabbitmq.Constants interface IMO

Reply:
`public static final AMQP.BasicProperties NO_PROPERTIES = new AMQP.BasicProperties();` would avoid the use of null here